app/main/helpers/service.py

```diff
@@ -0,0 +1,49 @@
+import re
```

Comment:
I take it everything in this helper is a straight lift from supplier app? (i.e. I'm not going to review it right now)